src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -37,17 +37,18 @@ internal enum TreeRotation
     [DebuggerTypeProxy(typeof(ICollectionDebugView<>))]
     [DebuggerDisplay("Count = {Count}")]
     [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public partial class SortedSet<T> : ISet<T>, ICollection<T>, ICollection, IReadOnlyCollection<T>, ISerializable, IDeserializationCallback
     {
         #region Local variables/constants
 
-        private Node _root;
-        private IComparer<T> _comparer;
-        private int _count;
-        private int _version;
+        private Node root; // Do not rename (binary serialization)
```

Comment:
I like these comments as a backup to the tests as in the past we've had holes in the serialization tests, it only needs one type missing or lost from the tests for the user scenario to break and we might not find out for a while after.